tests/core/test_timer.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Getting Things GNOME! - a personal organizer for the GNOME desktop
+# Copyright (c) 2008-2015 - Lionel Dricot & Bertrand Rousseau
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program.  If not, see <http://www.gnu.org/licenses/>.
+# -----------------------------------------------------------------------------
+
+import datetime
+
+from unittest import TestCase
+from mock import patch
+
+from GTG.core.timer import Timer
+
+
+class Config(object):
+    """Creating a mock Config
+    to be able to create a Timer object later
+    """
+    def __init__(self):
+        self.values = {'hour': '00', 'min': '00'}
+
+    def set(self, name, value):
+        self.values[name] = value
+
+    def get(self, name):
+        return self.values.get(name)
+
+
+class TestTimerParser(TestCase):
+
+    def setUp(self):
+        patch.object(Timer, 'connect_to_dbus').start()
+        self.config = Config()
+        self.timer = Timer(self.config)
+
+    def tearDown(self):
+        patch.stopall()
+
+    def test_time_parser_classic_format(self):
+        self.assertEqual(self.timer.parse_time('00:00'), datetime.time(00, 00))
```

Comment:
Please fix it. Somebody might later change it from 07 to 09 only to get an error. Keep it simple, stupid and use only one number-system